client/cjdroute2.c

```diff
@@ -730,13 +721,10 @@ int main(int argc, char** argv)
                         logger,
                         allocator);
 
-    // --------------------- noBackground ------------------------ //
-
-    int64_t* noBackground = Dict_getInt(&config, String_CONST("noBackground"));
-    if (forceNoBackground || (noBackground && *noBackground)) {
```

Comment:
The initial change in 4e0d8ad15a280a564b818c57b94277b5af8d797b was quite complex, but was ensuring backward compatibility. If you'd prefer to merge that instead, I'd clean up this branch to `--nobg`, `--bg` and removing it from `--genconf`.